DominantColor/Shared/DominantColors.swift

```diff
@@ -161,19 +161,19 @@ private func distanceForAccuracy(accuracy: GroupingAccuracy) -> (INVector3, INVe
     case .Low:
         return CIE76SquaredColorDifference
     case .Medium:
-        return CIE94SquaredColorDifference()
+        return { CIE94SquaredColorDifference()(colors: $0) }
```

Comment:
Thank you for the explanation. You can change those function signatures then :+1: